packages/@robojs/cron/src/core/cron.ts

```diff
@@ -76,9 +72,11 @@ class CronJob {
 				}
 				return jobs
 			},
-			{ namespace: '__plugin_cron_' }
+			{ namespace: NAMESPACE }
 		)
 
+		setState(`${NAMESPACE}${jobId}`, this)
```

Comment:
Would you mind also making this compatible with function-based Cron jobs by moving this earlier in the function and wrapping the rest in a conditional?

Since it currently throws an error for function-based jobs, perhaps that can be turned into a debug log and return early instead. That way, if anyone wonders why function-based jobs aren't persisting, they can see why in debug mode.